src/App.js

```diff
@@ -49,6 +53,8 @@ class App extends Component {
                height={item.height}     // Height - used for a CSS style to position on the screen
                layer={100 + i}          // Layer - used for a CSS style to show items on-top of bg
                key={item.id}            // Key - to help React with performance
+               type={item.type}
+               onClickForPoints={this.onItemClicked}
```

Comment:
I suggest naming your callback props with `Callback` in the name.  So something like `onClickforPointsCallback`